### DISCUSSION TITLE
Alienware X15 R1

### DISCUSSION THREAD

**Discussion by BobbyMarova** [T-Troll/alienfx-tools#213] [Ideas]
Here is the code for turning off alien head in the back and tron effect:

REM dev,light,r,g,b,br
REM TURN OFF ALIEN HEAD AT BACK dev 1 light 1
alienfx-cli.exe set-one=1,1,0,0,0,0
REM TURN OFF TRON EFFECT DEV 1 light 9 to 19
alienfx-cli.exe set-one=1,9,0,0,0,0
alienfx-cli.exe set-one=1,10,0,0,0,0
alienfx-cli.exe set-one=1,11,0,0,0,0
alienfx-cli.exe set-one=1,12,0,0,0,0
alienfx-cli.exe set-one=1,13,0,0,0,0
alienfx-cli.exe set-one=1,14,0,0,0,0
alienfx-cli.exe set-one=1,15,0,0,0,0
alienfx-cli.exe set-one=1,16,0,0,0,0
alienfx-cli.exe set-one=1,17,0,0,0,0
alienfx-cli.exe set-one=1,18,0,0,0,0
alienfx-cli.exe set-one=1,19,0,0,0,0

And oh boy was it a pain in the back side reverse-engineering it.

**Comment by T-Troll** [comment]
Hmm... Is it REALLY 10 lights in to the rest of tron?

It was less pain in case you use GUI first, define lights then create zones to group it. Zones can be used in CLI as well (`set-zone` command).
Also, you can combine CLI commands into one line - `alienfx-cli set-one=1,2,3,4 set-one=5,6,7,8`.
